### PR TITLE
[RFR] Last design fixes

### PR DESCRIPTION
**Pull request by eschaefer**

### CODE REVIEW

**Comment by eschaefer** [comment]
@nburkley can you review/merge this morning?

File: web/static/css/app.css [hunk 1]
```diff
@@ -372,7 +371,7 @@ a {
 .action__button--white {
   border-radius: 3px;
   background-color: white;
-  box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.24), 0 0 1px 0 rgba(0, 0, 0, 0.12);
+  box-shadow: 0px 1px 2px 1px rgba(0, 0, 0, .24), 0 0 1px 0 rgba(0, 0, 0, .12);
```
Comment (nburkley) [inline web/static/css/app.css:374]: Just curious how you decided on this? Just eyeballing it or was it exported from zeplin?

File: web/templates/article/edit.html.eex [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
     <%= link "Back", to: article_path(@conn, :index) %>
     <footer>
-      <p>Have feedback or questions? Contact us</p>
+      <p>Have feedback or questions? <a href="mailto:c​.hanke@de​.edenspiekermann​.com?subject=POSTBot%20Feedback%20or%20Question">Contact us<a/></p>
```
Comment (nburkley) [inline web/templates/article/edit.html.eex]: the links should all be using the phoenix `link` tag

**Review by nburkley — APPROVED** [review]
I'll update the links and merge the changes

**Comment by nburkley** [comment]
Also, why is this PR being merge to `master` and not `develop`?

File: web/templates/article/edit.html.eex (hunk 2 above)
Comment (nburkley) [reply web/templates/article/edit.html.eex]: @eschaefer either your editor or wherever you pull the address from is inserting some weird chars into the email address
It's showing up in my editor with extra chars.
screen shot 2016-10-17 at 11 11 54

File: web/static/css/app.css (hunk 1 above)
Comment (eschaefer) [reply web/static/css/app.css:374]: Old style was from Zeplin, but it didn't work in Safari :/

**Comment by eschaefer** [comment]
It's a (dirty) hotfix, so isn't `master` where that gets merged? The tweaks were supposed to get deployed asap onto the heroku app.

File: web/templates/article/edit.html.eex (hunk 2 above)
Comment (eschaefer) [reply web/templates/article/edit.html.eex]: Eek! Ok Atom didn't display those.

**Comment by nburkley** [comment]
fyi @eschaefer I've moved the feedback link into a shared partial

**Pull request merged**